assets/product-info.js

```diff
@@ -72,21 +72,39 @@ if (!customElements.get('product-info')) {
       handleOptionValueChange({ data: { event, target, selectedOptionValues } }) {
         if (!this.contains(event.target)) return;
 
-        const baseUrl = target.dataset.productUrl || this.inflightRequestProductUrl || this.dataset.url;
-        this.inflightRequestProductUrl = baseUrl;
+        this.resetProductFormState();
 
-        const productForm = this.productForm;
-        productForm?.toggleSubmitButton(true);
-        productForm?.handleErrorMessage();
+        // url of the product associated with the selected option value
+        const productUrl = target.dataset.productUrl || this.pendingRequestUrl || this.dataset.url;
+
+        let requestUrl, shouldSwapProduct, shouldFetchFullPage;
+        if (this.isFeaturedProduct) {
```

Comment:
When we're expecting the backend to work for feat. product ?

Reply:
It's kicking off this week